scripts/releaseUtils.ts

```diff
@@ -133,6 +133,13 @@ export function getVersionChoices(currentVersion: string) {
             value: inc('major')
           }
         ]
+      : currentAlpha
+      ? [
+          {
+            title: 'beta',
+            value: inc('patch') + '-beta.1'
```

Comment:
I think we should keep starting with `.0` so we get one internal beta to play a bit, and then we release `beta.1` to announce it.
```suggestion
            value: inc('patch') + '-beta.0'
```